### PR TITLE
Finished Auth

### PR DESCRIPTION
**Pull request by AustinKobayashi**
Finished auth. Requires an email in the firebase 'admins' database

### CODE REVIEW

**Review by Ciuffi — COMMENTED** [review]
Doesn't seem to be working for me.. I added myself to firebase but I keep getting permission denied when I try to login. Have any idea why?

**Review by Ciuffi — APPROVED** [review]
Okay works now, the UID in the admin collection has to be the same as in firebase authentication

File: pages/index.vue [hunk 1]
```diff
@@ -21,11 +21,16 @@ export default {
...
+        const res = await auth.signInWithPopup(this.provider)
+        if (!(await fireDb.isAdmin(res.user.email))) {
+          alert('You are not an admin')
+          return
+        }
         this.$router.push('/cms')
       } catch (e) {
-        console.log(e)
-        if (e.code === 'auth/web-storage-unsupported') {
+        console.log(e.code)
```
Comment (Eric1015) [inline pages/index.vue]: Maybe you don't need 'console.log'

**Review by Ciuffi — CHANGES_REQUESTED** [review]
Sorry just two more things, could you add a logout button and a redirect in index.vue to the CMS if you're already authenticated?

**Review by Ciuffi — APPROVED** [review]
very persistent

**Pull request merged**